### PR TITLE
Fix the bugs in the navbar

### LINKED ISSUE #38: Make the Navbar image as a link to the sack site
**Is your feature request related to a problem? Please describe.**
Currently, the navbar image has no link

**Describe the solution you'd like**
Make the Navbar image as a link to the sack site

**Are you working on this?**
Yes

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #38 

## Goals

Making the bugs in the navbar

## Approach

I made the bugs in the navbar

### Preview Link

https://yoshitharathnayake.github.io/sack-blog/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: _includes/navigation.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <nav class="navbar shadow-none fixed-top navbar-expand-lg" id="sectionsNav">
     <div class="container">
         <div class="navbar-translate">
-            <a class="navbar-brand" href="#">
-                <img src="{{ site.baseurl }}/assets/img/logo/logo.png">
+            <a target="_blank" class="navbar-brand" href="http://sack.edu.lk/">
+                <img src="{{ site.baseurl }}/assets/img/logo/nav-logo.svg">
```
Comment (jayasanka-sack) [inline _includes/navigation.html]: Let's open it within the same page.
```suggestion
            <a class="navbar-brand" href="http://sack.edu.lk/">
                <img src="{{ site.baseurl }}/assets/img/logo/nav-logo.svg">
```

Comment (YoshithaRathnayake) [reply _includes/navigation.html]: now @jayasanka-sack

File: _includes/navigation.html [hunk 2]
```diff
@@ -1,6 +1,17 @@
 <!-- Navbar -->
 <nav class="navbar navbar-default navbar-expand-lg fixed-top" color-on-scroll="100">
     <div class="container">
+        <div class="navbar-translate">
```
Comment (Piumal1999) [inline _includes/navigation.html:4]: Could you please reformat this part? Seems like a div tag is missing.

If you are using VSCode, you can select this newly added part and right-click to find the formatting option

Comment (YoshithaRathnayake) [reply _includes/navigation.html:4]: You are telling to format the code right Shift+Alt+F right?

Comment (YoshithaRathnayake) [reply _includes/navigation.html:4]: @Piumal1999

**Pull request closed without merging**